hack/dockerfiles/test.buildkit.Dockerfile

```diff
@@ -233,6 +233,7 @@ RUN chmod u+s /usr/bin/newuidmap /usr/bin/newgidmap \
 # We lock the root account by `passwd -l root`, so as to disable su completely.
 
 # tonistiigi/buildkit:rootless-base is a pre-built multi-arch version of rootless-base-internal https://github.com/moby/buildkit/pull/666#pullrequestreview-161872350
+# [DNM] the base image needs to be updated
```

Comment:
@tonistiigi could you update the image?

Reply:
docker.io/tonistiigi/buildkit:rootless-base@sha256:7e87da0f64e4987b4b6620f7b27b62d7178fc27964fcc3afeb8412f4231aa425

Reply:
thanks, updated Dockerfile